src/app/notifications/page.tsx

```diff
@@ -0,0 +1,70 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+import { User } from "lucide-react";
+import Navbar from "../_components/Navbar";
```

Comment:
I understand that you have worked on the Navbar as part of another PR, correct? When starting this new branch for the notifications page, did you rebase on the Navbar branch? Here is the reason why I ask: 
I don't believe it is good practice to rebase on feature branches, as that creates a bit of unstability in the repo. I believe that we want to always branch out from main, ensure that main is on its latest version, and work from there. Once you return to work, you want to fetch from the main remote and rebase on main to ensure that if any PR has been merged, you get those updates for when you push your local and try to merge there are no merge conflicts. 
The reason why I am saying all of this is because I can't see the Navbar component when checking out the ava/notifications-page branch. I believe that is because that lives in another PR and when looking at Git Graph I don't see a common history between your branches ava/notifications-page and ava/navbar-update. 
Because there is no Navbar component in this PR, when I try to run the app from this branch and move to /notifications as per the PR instructions, I get a `Module not found: Can't resolve '../_components/Navbar'` error. 
I might be missing something here given that I see the screenshot you added in the PR with the functioning page, so please feel free to clarify and call out if I am missing something!

Reply:
@FlaIespa I'm getting the same error. The easiest solution is probably to merge the PR where the NavBar is introduced, stash the changes on this branch, pull the latest from the main branch, and then pop the stash with the changes on this branch. Let me know if this makes sense!

Reply:
I believe that she won't need to stash changes, given that all the work has been committed and pushed. I would say then maybe just merge the Navbar into the main, pull the latest from the main in her local, and then rebase this branch on the main. What do you think @matviikotolyk? But I would also be up to trying your approach!

Reply:
Oh yeah, that works. In my head, the changes were still on Ava's local machine... It seems like the PR with the Navbar looks good, so we can probably go ahead and merge it into main. @FlaIespa

Reply:
I just reviewed and it looks good to me too. I would just wait for Ava to merge it. Even though there does not seem to be any outstanding changes that need to be made,  there is still one unresolved conversation, so it might be better to let her address it. What do you think?

Reply:
Totally, sounds good to me!

Reply:
I will resolve this so we can merge it! I rebased it on main and then force-pushed with lease.